src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -297,9 +299,16 @@ export class HoldRequest extends React.Component {
         (<div className="call-number">
           <span>Call number:</span><br />{selectedItem.callNumber}
         </div>) : null;
-    const deliveryLocations = this.props.deliveryLocations;
-    const isEddRequestable = this.props.isEddRequestable;
-    const allClosed = closedLocations.includes('');
+    let itemClosedLocations = closedLocations;
+    if ('isRecap' in selectedItem && selectedItem.isRecap) itemClosedLocations = itemClosedLocations.concat(recapClosedLocations);
```

Comment:
Aren't these two checks more or less the same? Seems inconsistent with the rest of the codebase so it looks more complicated

Reply:
I originally wrote the checks this way to avoid counting items without `isRecap` as being specifically non-recap items. But looking at `item.js` it seems that this property will always be set, so I've simplified it.